bin/bundle-platform

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
Why does this have to be in the main `bin/` dir? That means it would end up in `/opt/chefdk/bin`, yes (I forget if we make the symlinks from a pre-defined list or do a foreach...).